src/components/card/documentation/info.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StoryHeader, StoryCode, StoryCodeBlock } from '../../../../.storybook/style/storybook-info.styles';
+
+const Info = (
+  <div>
+    <p>A Card component.</p>
+
+    <StoryHeader>How to use the Card component:</StoryHeader>
+
+    <p>In your file</p>
+
+    <StoryCode padded>
+      {'import Card from "carbon-react/lib/components/card";'}
+    </StoryCode>
+
+    <p>To render a simple Card:</p>
+
+    <StoryCodeBlock padded>
+      {'<Card>'}
+      {'  content'}
+      {'<Card>'}
+    </StoryCodeBlock>
+
+    <p>CardRow and CardColumn components could be used to position the content.</p>
+    <p>CardFooter component could be used to render the Card Footer.</p>
+
+    <StoryCodeBlock padded>
+      {'<Card>'}
+      {'  <CardRow>'}
+      {'    <CardColumn>content1</CardColumn>'}
+      {'    <CardColumn>content2</CardColumn>'}
+      {'  </CardRow>'}
+      {'  <CardFooter>'}
+      {'    <CardColumn>footer content 1</CardColumn>'}
+      {'    <CardColumn>footer content 2</CardColumn>'}
+      {'  </CardFooter>'}
+      {'<Card>'}
```

Comment:
Shouldn't this closing tag be `</Card>` instead of `<Card>` ?

Reply:
fixed

Reply:
Thanks for fixing this in f636ed7 :thumbsup: